scripts/monorepo/buildTo.js

```diff
@@ -0,0 +1,9 @@
+const { spawnSync } = require('child_process');
+const lernaBin = require.resolve('lerna/cli.js');
+
+const argv = process.argv.slice(2);
+const [project, ...rest] = argv;
+
+spawnSync(lernaBin, ['run', 'build', '--scope', project, '--include-filtered-dependencies', '--stream', '--', ...rest], {
```

Comment:
What do all these options do?

Reply:
I'll add a comment. But basically this informs lerna to build a specific project as well as all its dependencies (filtered is just a fancy way of saying a subset of packages)